src/pages/security.tsx

```diff
@@ -0,0 +1,47 @@
+// TODO include into coverage before page release
+/* istanbul ignore file */
```

Comment:
Please write a test for this page. You can look at the existing pages for examples.